lib/ansible/modules/system/mount.py

```diff
@@ -20,9 +20,20 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-ANSIBLE_METADATA = {'status': ['preview'],
-                    'supported_by': 'core',
-                    'version': '1.0'}
+
+from ansible.module_utils.basic import AnsibleModule, get_platform
+from ansible.module_utils.ismount import ismount
+from ansible.module_utils.pycompat24 import get_exception
+from ansible.module_utils.six import iteritems
+from ansible.module_utils._text import to_native
+import os
```

Comment:
As long as you're doing the work in this file, I don't care about the rest of the style changes but the imports need to go back where you found them.  We've told you several times that the boilerplate ansible variables at the top of the modules are not used by the module code and therefore they should be treated the same as the other boilerplate headers to code.  From the module's POV, these are comments. They use python variables so that other code can easily pull them out of the file using the ast module.

(This is written down in the "New Files" section of https://github.com/ansible/proposals/issues/50 )